ipatests/test_ipalib/test_config.py

```diff
@@ -563,7 +563,7 @@ def test_finalize_core(self):
         # Test using DEFAULT_CONFIG:
         defaults = dict(constants.DEFAULT_CONFIG)
         (o, home) = self.finalize_core(None, **defaults)
-        assert list(o) == sorted(defaults)
+        assert set(o).issuperset(set(defaults))
```

Comment:
@martbab I tried to do that, but that also sets the `fips_mode` key for the env object, doesn't it? We want the `fips_mode` key to not be present if ipaplatform wasn't imported.